src/main/java/core/basesyntax/garage/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.garage;
+
+public class Excavator extends Machine{
```

Comment:
There should not be an empty line after the class declaration. Please remove the redundant empty line.